dot/build_spec.go

```diff
@@ -94,33 +96,43 @@ func BuildFromDB(path string) (*BuildSpec, error) {
 	tmpGen.Genesis.Raw = make(map[string]map[string]string)
 	tmpGen.Genesis.Runtime = make(map[string]map[string]interface{})
 
+	// BootstrapMailer should not return an error here since there is no URLs to connect to
+	disabledTelemetry, err := telemetry.BootstrapMailer(context.Background(), nil, false, nil)
```

Comment:
Maybe change to `context.TODO()` :thinking:

Reply:
Better suitable